src/SFA.DAS.ApprenticeFeedback.Web/Startup/AuthenticationStartup.cs

```diff
@@ -52,7 +52,6 @@ public static class AuthenticationStartup
             services.AddHttpContextAccessor();
             services.AddRazorPages(o => o.Conventions
                 .AuthorizeFolder("/"));
-                //.AllowAnonymousToPage("/Index"));
             services.AddRazorPages();
```

Comment:
Do we need allow anonymous to index for a start page or was that just a temporary hack?